### PR TITLE
Fix dumping logs with logexporter

### PR DESCRIPTION
**Pull request by wojtek-t**

### CODE REVIEW

File: cluster/log-dump/log-dump.sh [hunk 1]
```diff
@@ -365,10 +365,12 @@ function dump_nodes_with_logexporter() {
   done
 
   failed_nodes=()
-  for node in "${NON_LOGEXPORTED_NODES[@]:-}"; do
-    echo "Logexporter didn't succeed on node ${node}. Queuing it for logdump through SSH."
-    failed_nodes+=("${node}")
-  done
+  if [[ -n "${NON_LOGEXPORTED_NODES}" ]]; then
```
Comment (shyamjvs) [inline cluster/log-dump/log-dump.sh]: :- (better to have it)

File: cluster/log-dump/log-dump.sh [hunk 2]
```diff
@@ -365,10 +365,12 @@ function dump_nodes_with_logexporter() {
...
+    for node in "${NON_LOGEXPORTED_NODES[@]}"; do
```
Comment (shyamjvs) [inline cluster/log-dump/log-dump.sh]: Also here.

Comment (wojtek-t) [reply cluster/log-dump/log-dump.sh]: Having it here means that we are ssh-ing to all nodes again after successful logexporter run.

I don't know why it is so - it works differently on my desktop.

**Comment by wojtek-t** [comment]
Reapplying label after fixing unbounded value.

**Pull request merged**